recipes/sadisplay/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "sadisplay" %}
+{% set version = "0.4.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af67160f89123886ab42b247262862bfcde0a3c236229ecdd59de0a1e8e35d96
+
+build:
+  number: 0
```

Could this be a noarch package?
```suggestion
  number: 0
  noarch: python
```

---

Is noarch ok with entry points? If so, yes, I think the code is pure python.

---

Yup, I don't know the exact details but I think conda uses the `entry_points` metadata to recreate them for the current platform at install time.